web/src/components/auth/NeedsAccountModal.tsx

```diff
@@ -43,48 +45,51 @@ export const NeedsAccountModal = (): ReactElement => {
 
   return (
     <Dialog
+      fullWidth={false}
       open={showNeedsAccountModal}
-      maxWidth="xs"
+      maxWidth="sm"
       onClose={(): void => setShowNeedsAccountModal(false)}
       data-testid={"self-reg-modal"}
+      aria-labelledby="modal"
+      disableEnforceFocus={contextualInfo.isVisible}
     >
-      <DialogTitle sx={{ p: 5, paddingRight: 10 }}>
-        <Heading level={0} styleVariant="h3">
+      <DialogTitle id="modal" className="display-flex flex-row flex-align-center margin-top-1 break-word">
+        <Heading level={0} styleVariant="h2" className="padding-x-1 margin-0-override">
           {Config.selfRegistration.needsAccountModalTitle}
         </Heading>
         <IconButton
           aria-label="close"
+          className="margin-left-auto"
           onClick={(): void => setShowNeedsAccountModal(false)}
           sx={{
-            position: "absolute",
-            right: 10,
-            top: 12,
             color: "#757575",
           }}
         >
           <Icon className="usa-icon--size-4">close</Icon>
         </IconButton>
       </DialogTitle>
-
-      <DialogContent dividers={false} sx={{ paddingX: 5, paddingBottom: 3 }}>
-        <Content>{Config.selfRegistration.needsAccountModalBody}</Content>
-      </DialogContent>
-      <DialogContent dividers={false} sx={{ paddingX: 5 }}>
-        <Box>
-          <PrimaryButton isColor="primary" isFullWidthOnDesktop onClick={linkToAccountSetup}>
-            {Config.selfRegistration.needsAccountModalButtonText}
-          </PrimaryButton>
-        </Box>
+      <DialogContent sx={{ padding: 0 }} dividers>
+        <div className="padding-x-4 margin-bottom-4 margin-top-2" data-testid="modal-body">
+          <Content>{Config.selfRegistration.needsAccountModalBody}</Content>
+          <Box>
+            <div className="margin-top-3">
+              <PrimaryButton isColor="primary" isFullWidthOnDesktop onClick={linkToAccountSetup}>
+                {Config.selfRegistration.needsAccountModalButtonText}
+              </PrimaryButton>
+            </div>
+          </Box>
+        </div>
       </DialogContent>
 
-      <DialogContent dividers={true} sx={{ paddingX: 5 }}>
+      <DialogContent>
```

Comment:
[dust] My reading on [MUI's documentation](https://mui.com/material-ui/react-dialog/) is that there should only be one child `<DialogContent />` item. Is that your read on it too?

Reply:
discussed, no changes will be made